mk/build.mk

```diff
@@ -28,6 +28,10 @@ COREDNS_VERSION ?= v1.10.0
 COREDNS_TMP_DIRECTORY ?= $(BUILD_DIR)/coredns
 COREDNS_PLUGIN_CFG_PATH ?= $(TOP)/tools/builds/coredns/templates/plugin.cfg
 
+EBPF_GIT_REPOSITORY ?= https://github.com/kumahq/merbridge.git
+EBPF_GIT_BRANCH ?= main
```

Comment:
shouldn't we be explicitly tracking a specific version?

Reply:
at this point we are not directly mirroring merbridge tree, and the plan is to upstream the changes, so I would leave it as is, and if anything would change, we can pin the specific version. Please open the conversation back if you disagree